src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
I see, let's roll with `animate` for now and we can adjust as we go. Could you change the `update`-method in the game screen to be called `animate`, too?